pkg/network/tracer/tracer_linux_test.go

```diff
@@ -1368,21 +1363,16 @@ func iptablesWrapper(t *testing.T, f func()) {
 	// Init iptables rule to simulate packet loss
 	rule := "INPUT --source 127.0.0.1 -j DROP"
 	create := strings.Fields(fmt.Sprintf("-I %s", rule))
-	remove := strings.Fields(fmt.Sprintf("-D %s", rule))
 
+	state := testutil.IptablesSave(t)
 	createCmd := exec.Command(iptables, create...)
 	err = createCmd.Start()
 	assert.Nil(t, err)
 	err = createCmd.Wait()
 	assert.Nil(t, err)
 
 	defer func() {
-		// Remove the iptable rule
-		removeCmd := exec.Command(iptables, remove...)
-		err = removeCmd.Start()
-		assert.Nil(t, err)
-		err = removeCmd.Wait()
-		assert.Nil(t, err)
+		testutil.IptablesRestore(t, state)
```

Comment:
Might as well use `t.Cleanup` here as well, right?

Reply:
We can't for this one since we want this execute when `iptablesWrapper` returns, not at the end of the test.

Reply:
Ahh, okay makes sense - thanks for clarifying. Seems good to me, then.